sshtunnel.py

```diff
@@ -1101,6 +1101,18 @@ def _raise(self, exception=BaseSSHTunnelForwarderError, reason=None):
         else:
             self.logger.error(repr(exception(reason)))
 
+    def _get_socket(self):
+        '''
+        This function returns socket according to ip version of ssh_host address
+        '''
+        addrinfo = socket.getaddrinfo(self.ssh_host, self.ssh_port)
+        for info in addrinfo:
+            family = info[0]
+            socktype = info[1]
+            if socktype == socket.SOCK_STREAM:
+                return socket.socket(family, socktype)
+        raise socket.error('Addrinfo for {},{} doesn\'t contain SOCK_STREAM')
```

Comment:
It looks like there is a missed `format`. 
I don't think that's it's a good idea to use `socket.error`.

Look at the similar code inside the paramiko: https://github.com/paramiko/paramiko/blob/538caf5/paramiko/client.py#L203-L216

I think it's better to use it.